scripts/add_env_vars.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+#
+# Developer contact: taksjosephrefugio@gmail.com
+#
+
+
+####################
+# GLOBAL VARIABLES #
+####################
+BASHRC="$HOME/Downloads/fake_bashrc"
```

Comment:
Update this to default before merging to master
```suggestion
BASHRC="$HOME/.bashrc"
```